src/modules/components/proposal/proposalDataListItem/approvalThresholdResult/approvalThresholdResult.tsx

```diff
@@ -27,9 +29,9 @@ export const ApprovalThresholdResult: React.FC<IApprovalThresholdResultProps> =
                 <span className="text-primary-400">
                     {formatterUtils.formatNumber(approvalAmount, { format: NumberFormat.GENERIC_SHORT })}
                 </span>
-                <span>of</span>
+                <span>{copy.approvalThresholdResult.of}</span>
```

Comment:
Could we also do a `outOf` copy here? Like:
```
const formattedApprovalThreshold = {formatterUtils.formatNumber(approvalThreshold, { format: NumberFormat.GENERIC_SHORT })};

[...]

<span>{copy.approvalThresholdResult.outOf(formattedApprovalThreshold)</span>
```

Reply:
✅ https://github.com/aragon/ods/pull/221/commits/4284e4d6f87e2510798e8184d8d7d6444dc4d1f1